lib/Parse/Lexer.cpp

```diff
@@ -1655,7 +1660,7 @@ static void validateMultilineIndents(const Token &Str,
 /// lexStringLiteral:
 ///   string_literal ::= ["]([^"\\\n\r]|character_escape)*["]
 ///   string_literal ::= ["]["]["].*["]["]["] - approximately
-void Lexer::lexStringLiteral() {
+void Lexer::lexStringLiteral(bool RawString) {
   const char *TokStart = CurPtr-1;
```

Comment:
Could you make `TokStart` points `r` for raw string literal?

Reply:
Hi @rintaro, that can be arranged but the proposal is meeting a bit of a head wind on swift-evolution so I think I‘ll wait to see if it is accepted before finalising the implementation.